Princeton-CDH/lenape-timetree
# make leaf layout forces easier to configure

to test out adjusting forces locally:
- Install hugo, clone this repository, and setup a local instance of this site following the setup instructions in the readme; more detailed setup instructions have been added to as part of the image branch, which you can access at https://github.com/Princeton-CDH/lenape-timetree/tree/feature/leaf-images-1#hugo-setup until #60 is merged into develop
- checkout this branch
- Run a local dev site with `hugo serve` and load the local site in your browser
- While the hugo server is running, edit the values for `forceStrength` in `assets/js/timetree.js` ; Hugo should automatically reload and apply the changes to these forces

For the purposes of reviewing this PR, it's sufficient to confirm that you can successfully edit the forces; we don't need to change them or figure out the right values before merging.

I have questions about priority of possible next steps for improving the layout. These are things I'm considering, you may think of others:
- allow several leaves for a single century+branch to spread out from each other more (i.e., add more pseudo nodes for branch+century combinations with more than 3 or 4 leaves)
-  make leaf labels wrap / multi-line (seems to require using `<tspan>`)
